api/src/Service/ServiceInterface.php

```diff
@@ -0,0 +1,28 @@
+<?php declare(strict_types=1);
+/*.
+    require_module 'standard';
+.*/
+
+namespace App\Service;
+
+interface ServiceInterface
+{
+
+
+    public function post(/*.mixed.*/$data): void;
```

Comment:
I know we're trying to line these up with POST, PUT, GET, DELETE, etc. API operations, but it might be nice if we go with `create` here instead of `post`.